jwst/lib/set_telescope_pointing.py

```diff
@@ -693,22 +807,30 @@ def calc_transforms(pointing, siaf, fsmcorr_version='latest', fsmcorr_units='arc
 
     Parameters
     ----------
-    pointing : Pointing
-        Observatory pointing information
+    t_pars : TransformParameters
+        The transformation parameters. Parameters are updated during processing.
 
-    siaf : SIAF
-        Aperture information
+    Returns
+    -------
+    transforms: Transforms
+        The list of coordinate matrix transformations
+    """
+    t_pars.method = t_pars.method if t_pars.method else Methods.default
+    transforms = t_pars.method.func(t_pars)
+    return transforms
 
-    fsmcorr_version : str
-        The version of the FSM correction calculation to use.
-        See :ref:`calc_sifov_fsm_delta_matrix`
 
-    fsmcorr_units : str
-        Units of the FSM correction values. Default is 'arcsec'.
-        See :ref:`calc_sifov_fsm_delta_matrix`
+def calc_transforms_original(t_pars: TransformParameters):
+    """Calculate transforms from pointing to SIAF using the original, pre-JSOCINT-555 algorithm
```

Comment:
Not sure what "from pointing to SIAF" is supposed to mean, exactly. Makes it sound like there's a transform that goes from pointing to SIAF. Is that what's intended? Isn't it kind of the other way around, in that pointing is computed, at least in part, *from* SIAF info, as opposed to *to* SIAF?

Reply:
✅ and updated the docstrings of the related functions